File: app/src/main/java/com/ps/simplepapersoccer/Activities/HowToPlayActivity.java
package com.ps.simplepapersoccer.Activities;

import android.os.Bundle;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.ps.simplepapersoccer.R;

public class HowToPlayActivity extends AppCompatActivity {

    ImageView leftArrow;
    ImageView rightArrow;
    TextView viewPagerText;
    private int[] sliderTextId = new int[]{ R.string.tutorial1_text, R.string.tutorial2_text, R.string.tutorial3_text, R.string.tutorial4_text };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_how_to_play);

        getSupportActionBar().setDisplayShowHomeEnabled(true);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        leftArrow = (ImageView)findViewById(R.id.left_nav);
        rightArrow = (ImageView)findViewById(R.id.right_nav);

        viewPagerText = (TextView) findViewById(R.id.viewpagerText);

        leftArrow.setVisibility(View.INVISIBLE);

        final ViewPager mViewPager = (ViewPager)findViewById(R.id.viewpager);
        TutorialPagerAdapter adapterView = new TutorialPagerAdapter(this, this);
        mViewPager.setAdapter(adapterView);

        mViewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
            @Override
            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {

            }

            @Override
            public void onPageSelected(int position) {
                viewPagerText.setText(getString(sliderTextId[position]));
                if (position == 0)
                {
                    leftArrow.setVisibility(View.INVISIBLE);
                    rightArrow.setVisibility(View.VISIBLE);
                }
                else if (mViewPager.getAdapter().getCount() - 1 == position){
                    leftArrow.setVisibility(View.VISIBLE);
                    rightArrow.setVisibility(View.INVISIBLE);
                }
                else {
                    leftArrow.setVisibility(View.VISIBLE);
                    rightArrow.setVisibility(View.VISIBLE);
                }
            }

            @Override
            public void onPageScrollStateChanged(int state) {

            }
        });
    }

    @Override
    public boolean onSupportNavigateUp(){
        finish();
        return true;
    }
}

File: app/src/main/java/com/ps/simplepapersoccer/Activities/HowToPlayActivity.kt
package com.ps.simplepapersoccer.activities

import android.os.Bundle
import androidx.viewpager.widget.ViewPager
import androidx.appcompat.app.AppCompatActivity
import android.view.View
import com.ps.simplepapersoccer.R
import kotlinx.android.synthetic.main.activity_how_to_play.*

class HowToPlayActivity : AppCompatActivity() {
    private val sliderTextId = intArrayOf(R.string.tutorial1_text, R.string.tutorial2_text, R.string.tutorial3_text, R.string.tutorial4_text)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_how_to_play)

        left_nav?.visibility = View.INVISIBLE

        val mViewPager = findViewById<ViewPager>(R.id.viewpager)
        val adapterView = TutorialPagerAdapter(this)
        mViewPager.adapter = adapterView

        mViewPager.addOnPageChangeListener(object : ViewPager.OnPageChangeListener {
            override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) {}

            override fun onPageSelected(position: Int) {
                viewpagerText?.text = getString(sliderTextId[position])
                when (position) {
                    0 -> {
                        left_nav?.visibility = View.INVISIBLE
                        right_nav?.visibility = View.VISIBLE
                    }
                    (mViewPager.adapter?.count ?: 0) - 1 -> {
                        left_nav?.visibility = View.VISIBLE
                        right_nav?.visibility = View.INVISIBLE
                    }
                    else -> {
                        left_nav?.visibility = View.VISIBLE
                        right_nav?.visibility = View.VISIBLE
                    }
                }
            }

            override fun onPageScrollStateChanged(state: Int) {}
        })
    }

    override fun onSupportNavigateUp(): Boolean {
        finish()
        return true
    }
}
